src/main/java/bio/terra/service/dataset/flight/ingest/IngestLandingFileDeleteGcpStep.java

```diff
@@ -0,0 +1,36 @@
+package bio.terra.service.dataset.flight.ingest;
+
+import bio.terra.service.filedata.google.gcs.GcsPdao;
+import bio.terra.stairway.FlightContext;
+import bio.terra.stairway.Step;
+import bio.terra.stairway.StepResult;
+import bio.terra.stairway.exception.RetryException;
+
+public class IngestLandingFileDeleteGcpStep implements Step {
```

Comment:
it sounds like we're calling the feature PayloadIngest, so maybe add that to the name to make it easy to tell it's part of the feature?

Reply:
That's interesting. I figured in the context of a flight it might be too vague